test/partially_blindrsa.test.ts

```diff
@@ -124,20 +124,24 @@ describe.each(vectors)('Errors-vec$#', (v: Vector) => {
     });
 });
 
-test.each(vectors)('TestVector_$#/safePrimes', (v: Vector) => {
-    prepare_sjcl_random_generator();
-    expect(isSafePrime(new sjcl.bn(v.p))).toBe(true);
-    expect(isSafePrime(new sjcl.bn(v.q))).toBe(true);
-});
-
-describe.each(vectors)('TestVector_$#', (v: Vector) => {
+test.each(vectors)(
+    'TestVector_%#/safePrimes',
+    (v: Vector) => {
+        prepare_sjcl_random_generator();
+        expect(isSafePrime(new sjcl.bn(v.p))).toBe(true);
+        expect(isSafePrime(new sjcl.bn(v.q))).toBe(true);
+    },
+    { timeout: 60_000 },
```

Comment:
From CI:

> Using an object as a third argument is deprecated. Vitest 4 will throw an error if the third argument is not a timeout number. Please use the second argument for options. See more at 

https://vitest.dev/guide/migration#test-options-as-a-third-argument


pass only the number in the third argument.